doc/api/url.md

```diff
@@ -131,7 +131,7 @@ return `true`.
 
 * `input` {string} The absolute or relative input URL to parse. If `input`
   is relative, then `base` is required. If `input` is absolute, the `base`
-  is ignored.
+  is ignored. If `input` is not a string, it is [converted to a string][] first.
 * `base` {string|URL} The base URL to resolve against if the `input` is not
```

Comment:
`base` is also `USVString` so perhaps this should be `{string}` rather than `{string|URL}` and also include the note about being converted to a string that is added on line 134 for `input`.

Reply:
> `base` is also `USVString` so perhaps this should be `{string}` rather than `{string|URL}` and also include the note about being converted to a string that is added on line 134 for `input`.

@TimothyGu @jasnell Sound OK to you?